backend/billing/views.py

```
@@ -0,0 +1,709 @@
+                'custom_id': f'{request.user.id}:{plan_type}:{billing_cycle}',
+            }],
+            'application_context': {
+                'brand_name': 'Trade Scan Pro',
+                'landing_page': 'NO_PREFERENCE',
+                'user_action': 'PAY_NOW',
+                'return_url': f'{settings.FRONTEND_URL}/checkout/success',
+                'cancel_url': f'{settings.FRONTEND_URL}/checkout/failure',
+            }
+        }
+        
+        response = requests.post(
```

Ruff (0.14.3)

Probable use of `requests` call without timeout

(S113)